javascript/javascript1/week4/h4.js

```diff
@@ -0,0 +1,97 @@
+let userName = "";
+const toDoList = [];
+
+function getReply(command) {
+    if (typeof command !=='string' || command === "") {
+        return `Error! Not a valide command`;
+    } if (command === "Hello my name is Violeta" && userName === "") {
+        userName = "Violeta";
+        return `Nice to meet you ${userName}`;
+    } else if (command === "Hello my name is Violeta" && userName === ""){
+        userName="";
+        return `Your name is already added`;
+    } if (command === "What is your name?" && userName != "") {
+        return `Your name is ${userName}`;
+   
+    }  else if ( command === "What is your name" && userName === "") {
+        return 'Your name is not mentioned';
```

Comment:
Something like this, your solution doesn't work when `userName` is `""`
```suggestion
  if (command === "What is your name?") {
    	if (userName) {
            return `Your name is ${userName}`;
        } else  {
           return "Your haven't mentioned your name";
        }
    }  
```

Reply:
Be careful if you use `""` or `''`, choose one, don't combine. I think I wrote about it previously. 🙂